exercises/practice/grade-school/grade-school-test.rkt

```diff
@@ -0,0 +1,195 @@
+#lang racket/base
+
+(require racket/class)
+(require "grade-school.rkt")
+
+(module+ test
+  (require rackunit rackunit/text-ui))
+
+(module+ test
+  (define suite
+    (test-suite "grade school tests"
+      (test-case "Roster is empty when no student is added"
+	(define school (new school%))
+        (check-equal? (send school roster) '()))
+
+      (test-case "Add a student"
+	(define school (new school%))
+        (check-true (send school add "Aimee" 2)))
+
+      (test-case "Student is added to the roster"
+	(define school (new school%))
+        (check-true (send school add "Aimee" 2))
+	(check-equal?
+          (send school roster)
+	  '((2 "Aimee"))))
+
+      (test-case "Adding multiple students in the same grade in the roster"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-true (send school add "Paul" 2)))
+
+      (test-case "Multiple students in the same grade are added to the roster"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-true (send school add "Paul" 2))
+        (check-equal?
+	  (send school roster)
+	  '((2 "Blair" "James" "Paul"))))
+
+      (test-case "Cannot add student to same grade in the roster more than once"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-false (send school add "James" 2))
+        (check-true (send school add "Paul" 2)))
+
+      (test-case "A student can't be in two different grades"
```

Comment:
This test case is c125dab7-2a53-492f-a99a-56ad511940d8 which is reimplemented by the following test case (a0c7b9b8-0e89-47f8-8b4a-c50f885e79d1). Therefore, we should remove this test as well.